### DISCUSSION TITLE
Export statistics are gone?

### DISCUSSION THREAD

**Discussion by TB-HunBoka** [dbeaver/dbeaver#22833] [Q&A] [answered]
When I export a data set from Oracle DB, as I remember there was a "progress bar" at the bottom of the Data Transfer object.
![image](https://github.com/dbeaver/dbeaver/assets/126617861/b3c3fbc1-276a-4921-a73c-2cad5c462b6c)
Now it says "Waiting for jobs to finish" and I have to open the background tasks tab to see the progress of the export. I saw some recent changes on the export functionality. I am using the Version 23.3.4.202402060628 version.
What I would like to ask, did I change it with a setting that I don't remember, or is this the expected behavior?

**Comment by LonwoLonwo** [comment, marked as answer]
Thanks for noticing. We created a ticket from this discussion.
https://github.com/dbeaver/dbeaver/issues/22879
